capstone/src/main/java/com/google/sps/Objects/Group.java

```diff
@@ -6,15 +6,15 @@
 public final class Group {
 
   private final ArrayList<String> memberIds;
-  private final ArrayList<Challenge> challenges;
-  private final ArrayList<Post> posts;
-  private final ArrayList<Option> options;
+  private final ArrayList<Long> challenges;
```

Comment:
changes these to challengeIds, postIds, and optionIds might help make it clearer what these are

Reply:
I think that makes sense too, but I've started storing them in datastore as challenges, posts, options. I could leave them in datastore like that which might cause confusion since they're named differently than the object properties, or I can change everything, which could affect things in multiple different servlets. what do you think is best?